BAZProjectC1/View/DashBoard/Home/HomeViewController.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  HomeViewController.swift
+//  BAZProjectC1
+//
+//  Created by 1044336 on 21/09/22.
+//
+
+import UIKit
+
+class HomeViewController: UIViewController {
+
+    let filterDataArray = ["Trending","Now Playing","Popular","Top Rated","Upcoming"]
```

Comment:
Cuida el espaciado:

Busquemos en el futuro mover esta lista a un modelo de datos.
```suggestion
    let filterDataArray = ["Trending", "Now Playing", "Popular", "Top Rated", "Upcoming"]
```